packages/accounts-2fa/2fa-server.js

```diff
@@ -124,6 +124,9 @@ Meteor.methods({
     );
   },
   has2faEnabled(selector) {
+    if (!Meteor.user()) {
+      throw new Meteor.Error(400, 'No user logged in.');
```

Comment:
Have you checked if 400 is a pattern on other accounts packages?

Reply:
It seems to be, at least on the `accounts-password`. Example: https://github.com/meteor/meteor/blob/f9ce040fc3d5c0e92fbb9b00630bcbc25d0d2ffd/packages/accounts-password/password_client.js#L147-L149